src/common/users.ts

```diff
@@ -377,6 +378,13 @@ export const clearUserStreak = async (
   con: DataSource | EntityManager,
   userIds: string[],
 ): Promise<number> => {
+  const events = userIds.map((userId) => ({
```

Comment:
Are we sure all these users are being affected before we send the event?
Should we consider moving it to CDC?

Reply:
I mean here the query will always set.
We do checks before to see it needs to be cleared or is not 0.